packages/vat-data/src/index.js

```diff
@@ -42,9 +42,9 @@ export {
   prepareSingleton,
 } from './exo-utils.js';
 
-/** @typedef {import('@agoric/swingset-liveslots').Baggage} Baggage */
 /** @typedef {import('@agoric/swingset-liveslots').DurableKindHandle} DurableKindHandle */
 /** @template T @typedef {import('@agoric/swingset-liveslots').DefineKindOptions<T>} DefineKindOptions */
+/** typedef {import('@agoric/swingset-liveslots').Baggage} Baggage */
```

Comment:
removing the `@` from `@typedef` was on purpose? Another problem with aliasing `Baggage`?

Reply:
oh, I need to come back to this. Probably explain why it's there. I should file an issue too (on typedoc) and reference it from this line